app/assets/images/administrate/search.svg

```diff
@@ -1,6 +1,6 @@
-<svg width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
+<svg id="search" width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you clarify the need for this addition of an `id`? If it’s for styles, we should use a class.